### PR TITLE
Make the docs logo readable on light backgrounds

### PR DESCRIPTION
**Pull request by rzadp**
Currently, the logo on the docs site blends into the background on the light theme:

[Here](https://internals.rust-lang.org/t/feedback-on-new-rust-documentation-logo-background/12787) is a discussion related to this. There doesn't seem to be a perfect way to handle this situation.

---

Silent because the docs crate is not part of the regular release process.

### CODE REVIEW

**Review by kianenigma — APPROVED** [review]
What I have anticipated for now is to always set the theme to something dark by default; this is part of the build headers. 

How does it look now in all themes?

**Comment by rzadp** [comment]
> How does it look now in all themes?

(disregard the highlighted `polkadot_sdk_docs`, I was hovering over it)

## Before




## After

**Comment by rzadp** [comment]
> What I have anticipated for now is to always set the theme to something dark by default; this is part of the build headers.

Yes, I see that dark `ayu` [is set by default](https://github.com/paritytech/polkadot-sdk/blob/fd522b818693965b6ec9e4cfdb4182c114d6105c/.gitlab/pipeline/build.yml#L94), however for me it seems to default to `system preference` anyway, which is light during the day. Seem like rustdoc overrides the default theme, if a system preference exists?

**Comment by rzadp** [comment]
Recreated the PR using the logos from github. No more tricks with shadow.

Screenshot 2024-08-26 at 12 26 05

Screenshot 2024-08-26 at 12 26 18

**Review by mordamax — APPROVED** [review]

File: docs/sdk/assets/theme.css [hunk 1]
```diff
@@ -9,9 +9,15 @@
...
 		width: 190px;
+		height: 52px;
 	}
 
 	nav.sidebar {
 		flex: 0 0 250px;
 	}
 }
+
+/* A shadow for the logo to be readable on a light background */
```
Comment (ordian) [inline docs/sdk/assets/theme.css:20]: is this comment outdated now?
```suggestion
```

**Review by ordian — APPROVED** [review]

**Pull request merged**